### DISCUSSION TITLE
Option for No Music? (Cannonball Outrun)

### DISCUSSION THREAD

**Discussion by sega-f** [djyt/cannonball#141] [Ideas]
Hi, i know the music in outrun is great but would be good to have the option to play with the music switched off, or even better, to have music volume control, for example 012345 (with zero being 'music off' and five being music full volume).
Could this be done? ....and btw, thanks i'm loving Cannonball Outrun, it runs so smooth it's frickin' awesome

**Comment by GallopRacer** [comment]
Just add a custom bgm as an empty wave.

**Reply by sega-f** [reply]
thanks that's a damn good idea but tbh i can't understand how to add custom tracks, there doesn't seem to be a clear guide to follow. . . .Any chance you could do a youtube video showing the step-by-step process for adding custom music?

**Reply by MichaelRPowell** [reply]
`You must edit the config.xml file to configure the music. You can see some examples already setup in the configuration.`

Basically get a WAV of the music you want (silence) and put it in /res/ folder. Then change the XML to have a name like "No Music" and point it to the EmptyWav.wav.


`	
	<custom_music>
		<track1 enabled="0">
			<title>RADIATION</title>
			<filename>switch_radiation.bin</filename>
		</track1>
		<track2 enabled="0">
			<title>CRUISING LINE</title>
			<filename>switch_cruisingline.bin</filename>
		</track2>
		<track3 enabled="0">
			<title>CAMINO A MI AMOR</title>
			<filename>switch_camino.bin</filename>
		</track3>
		<track4 enabled="0">
			<title>STEP ON BEAT</title>
			<filename>switch_steponbeat.bin</filename>
		</track4>
		<track5 enabled="0">
			<title>STEP ON BEAT MEGADRIVE</title>
			<filename>switch_steponbeat_md.bin</filename>
		</track5>
		<track6 enabled="0">
			<title>MIDNIGHT HIGHWAY</title>
			<filename>switch_midnighthighway.wav</filename>
		</track6>
	</custom_music>`

So you would have:

`
		<track7 enabled="1">
			<title>No Music</title>
			<filename>emptywav.wav</filename>
		</track7>
`

**Reply by sega-f** [reply]
thanks so much man, i will follow this guide. You're the best!
